examples/research_projects/sd3_lora_colab/train_dreambooth_lora_sd3_miniature.py

```diff
@@ -765,7 +765,7 @@ def load_model_hook(models, input_dir):
         lora_state_dict = StableDiffusion3Pipeline.lora_state_dict(input_dir)
 
         transformer_state_dict = {
-            f'{k.replace("transformer.", "")}': v for k, v in lora_state_dict.items() if k.startswith("unet.")
+            f'{k.replace("transformer.", "")}': v for k, v in lora_state_dict.items() if k.startswith("transformer.")
```

Comment:
Could use `StableDiffusion3Pipeline.transformer_name` to be sure. But this is okay for now.